lib/view_component/system_test_helpers.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module ViewComponent
+  module SystemTestHelpers
+    include TestHelpers
+    include Capybara::DSL
+
+    def page
+      Capybara.current_session
+    end
+
+    def with_rendered_component_in_browser(component, **options, &block)
+      layout = options[:layout] || false
+
+      opts = {
+        layout: layout,
+        locals: {
+          render_args: {
+            component: component,
+            template: "view_components/preview",
```

Comment:
Do we need this? We're already referencing the template on line 25.

Reply:
Yep not necessary, it looks like this is specifically used when component is not provided. I'll remove.